src/fragments/lib/pubsub/js/connection-states.mdx

```diff
@@ -0,0 +1,10 @@
+#### Connection states
+
+- **Connected** - The service is connected and working with no issues.
+- **ConnectedPendingDisconnect** - The service connection has no active subscriptions and is disconnecting.
+- **ConnectedPendingKeepAlive** - The service connection is open, but has missed expected keep alive messages.
+- **ConnectedPendingNetwork** - The service connection is open, but the network connection has been disrupted. When the network recovers, the connection will continue serving traffic.
+- **Connecting** - The service is attempting to connect.
+- **ConnectionDisrupted** - The service connection is disrupted and the network is available.
+- **ConnectionDisruptedPendingNetwork** - The service connection is disrupted and the network connection is unavailable.
+- **Disconnected** - The service has no active subscriptions and has been disconnected.
```

Comment:
I would do **`Connected`** to highlight that is `code`, same for the others

Reply:
Updated. Didn't make as clear a difference as I would have expected